king_phisher/server/fs_utilities.py

```diff
@@ -56,6 +58,72 @@ def chown(path, user=None, group=constants.AUTOMATIC, recursive=True):
 	:type group: int, str, ``None``, :py:class:`~king_phisher.constants.AUTOMATIC`
 	:param bool recursive: Whether or not to recurse into directories.
 	"""
+
+	user, group = _resolve_information(user, group)
+
+	if recursive:
+		iterator = smoke_zephyr.utilities.FileWalker(path)
+	else:
+		iterator = (path,)
+	for path in iterator:
+		os.chown(path, user, group)
+
+
+def access(path, mode, user=None, group=constants.AUTOMATIC):
+	"""
+	This is a high-level wrapper around :py:func:`os.access` to provide
+	additional functionality. Similar to `os.access` this high-level wrapper
+	will test the given path for a variety of access modes. Additionally however,
+	*user* or *group* can be specified to test against a specific of user or group.
+
+	.. versionadded:: 1.14.0
+
+	:param str path: The path to test access for.
+	:param str mode: The mode to test access for. Set to `R_OK` to test for readability,
```

Comment:
So mode should be an integer and should support multiple flag values like `os.R_OK | os.W_OK` instead of being a string.